UPGRADING.md

```diff
@@ -0,0 +1,105 @@
+# Upgrading
+
+This section describes how to upgrade a Conjur Server.
+
+## Standard Process
+
+The following steps describe a standard upgrade of the Conjur server, when deployed
+using Docker Compose. These steps assume you have defined your Conjur image in
+a service named `conjur`, and that you have access to the Conjur data key
```

Comment:
The mention of Conjur data key is much more important than naming of `conjur` service
Can we put the spotlight on it? It can be confusing with API KEY I think we need to give here a link ([maybe from here](https://www.conjur.org/blog/loading-your-database-credentials-at-runtime-with-conjur/) or [here](https://github.com/cyberark/conjur/blob/1d2e0c3b01be7a91bb30381c4b895b0664801537/README.md#important-avoid-data-loss) ) to where we suggsted to store it, that it's a Base64-encoded 256 bit data encrytion key etc, without it - there is no upgrade